pkg/cmd/kind/delete/cluster/deletecluster.go

```diff
@@ -39,10 +39,15 @@ func NewCommand(logger log.Logger, streams cmd.IOStreams) *cobra.Command {
 	flags := &flagpole{}
 	cmd := &cobra.Command{
 		Args: cobra.NoArgs,
-		// TODO(bentheelder): more detailed usage
 		Use:   "cluster",
 		Short: "Deletes a cluster",
-		Long:  "Deletes a resource",
+		Long:  `Deletes a Kind cluster from the system.
+
+This is an idempotent operation, meaning it may be called multiple times without
+failing. If the cluster exists it will be deleted, and if the cluster is already
+gone it will just return success. Errors will only occur if the cluster exists
+and was not able to be deleted.
```

Comment:
```suggestion
This is an idempotent operation, meaning it may be called multiple times without
failing (like `rm -f`). If cluster resources exist they will be deleted, and if the cluster is already
gone it will just return success. Errors will only occur if cluster resource exist
and are not able to be deleted.
```